package.json

```diff
@@ -29,7 +29,7 @@
     "publish-beta": "learna run build && npm run test && npm run test-dist && scripts/publish.sh beta",
     "test": "npm run lint && npm run test-node && npm run test-browser && npm run test-render-react",
     "test-fast": "npm run test-node",
-    "test-ci": "node test/start.js test-ci && npm run build && npm run collect-metrics",
+    "test-ci": "node test/start.js test-ci && npm run collect-metrics",
```

Comment:
Does this ensure there is a build to measure, or how does that happen?

Reply:
I'm putting it back